src/components/ProjectCard/ProjectCard.css

```diff
@@ -0,0 +1,45 @@
+.ProjectCard {
+	display: block;
+	max-width: 230px;
+	padding: 25px;
+	margin-bottom: 15px;
+	margin-right: 15px;
+	background-color: #fff;
+	border: 1px solid #ddd;
+	border-radius: 2px;
+	box-shadow: 0 2px 5px 0 rgba(0, 0, 0, .16), 0 2px 10px 0 rgba(0, 0, 0, .12);
+	text-decoration: none !important;
+	color: inherit;
+	-moz-transition: all, .15s;
+	-o-transition: all, .15s;
+	-webkit-transition: all, .15s;
```

Comment:
__*Preference*__
This requires a double check here, but I believe CRA applies autoprefixer. So you can get rid of the vendor specific tags here.

Reply:
I tried removing a few, and they didn't show up in the build version, so I'm going to leave it for now.